src/testcafe.js

```diff
@@ -42,10 +45,18 @@ export default class TestCafe {
     }
 
     createRunner () {
-        return new Runner(this.proxy, this.browserConnectionGateway);
+        var newRunner = new Runner(this.proxy, this.browserConnectionGateway);
+
+        this.runners.push(newRunner);
+        promisifyEvent(newRunner, 'stopped').then(() => remove(this.runners, newRunner));
+
+        return newRunner;
     }
 
-    close () {
+    async close () {
+        await Promise.all(this.runners.slice().map(runner => runner.stop()));
```

Comment:
@inikulin I suppose we should not force stopping of all current runners in the `testcafe.close()` method. We should only close opened `browserConnections` (to clear heartbeat timeouts) and close the `proxy` to avoid node process hanging. Or I'm wrong?